modules/Gallery2.js

```diff
@@ -0,0 +1,49 @@
+const Gallery2 = () =>
```

Comment:
Hi @victor-akinboro, what's the difference between Gallery2 and Gallery1?

Reply:
they have the same structure but different content in them